lib/src/generated/cosmos/authz/v1beta1/authz.pb.dart

```diff
@@ -28,43 +39,70 @@ class GenericAuthorization extends $pb.GeneratedMessage {
     }
     return _result;
   }
-  factory GenericAuthorization.fromBuffer($core.List<$core.int> i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromBuffer(i, r);
-  factory GenericAuthorization.fromJson($core.String i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromJson(i, r);
-  @$core.Deprecated(
-  'Using this can add significant overhead to your binary. '
-  'Use [GeneratedMessageGenericExtensions.deepCopy] instead. '
-  'Will be removed in next major version')
-  GenericAuthorization clone() => GenericAuthorization()..mergeFromMessage(this);
-  @$core.Deprecated(
-  'Using this can add significant overhead to your binary. '
-  'Use [GeneratedMessageGenericExtensions.rebuild] instead. '
-  'Will be removed in next major version')
-  GenericAuthorization copyWith(void Function(GenericAuthorization) updates) => super.copyWith((message) => updates(message as GenericAuthorization)) as GenericAuthorization; // ignore: deprecated_member_use
+  factory GenericAuthorization.fromBuffer($core.List<$core.int> i,
+          [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) =>
+      create()..mergeFromBuffer(i, r);
+  factory GenericAuthorization.fromJson($core.String i,
+          [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) =>
+      create()..mergeFromJson(i, r);
+  @$core.Deprecated('Using this can add significant overhead to your binary. '
+      'Use [GeneratedMessageGenericExtensions.deepCopy] instead. '
+      'Will be removed in next major version')
+  GenericAuthorization clone() =>
+      GenericAuthorization()..mergeFromMessage(this);
+  @$core.Deprecated('Using this can add significant overhead to your binary. '
+      'Use [GeneratedMessageGenericExtensions.rebuild] instead. '
+      'Will be removed in next major version')
+  GenericAuthorization copyWith(void Function(GenericAuthorization) updates) =>
+      super.copyWith((message) => updates(message as GenericAuthorization))
+          as GenericAuthorization; // ignore: deprecated_member_use
   $pb.BuilderInfo get info_ => _i;
   @$core.pragma('dart2js:noInline')
   static GenericAuthorization create() => GenericAuthorization._();
   GenericAuthorization createEmptyInstance() => create();
-  static $pb.PbList<GenericAuthorization> createRepeated() => $pb.PbList<GenericAuthorization>();
+  static $pb.PbList<GenericAuthorization> createRepeated() =>
+      $pb.PbList<GenericAuthorization>();
   @$core.pragma('dart2js:noInline')
-  static GenericAuthorization getDefault() => _defaultInstance ??= $pb.GeneratedMessage.$_defaultFor<GenericAuthorization>(create);
+  static GenericAuthorization getDefault() => _defaultInstance ??=
+      $pb.GeneratedMessage.$_defaultFor<GenericAuthorization>(create);
   static GenericAuthorization? _defaultInstance;
 
   @$pb.TagNumber(1)
   $core.String get msg => $_getSZ(0);
   @$pb.TagNumber(1)
-  set msg($core.String v) { $_setString(0, v); }
+  set msg($core.String v) {
```

Comment:
does "v" stand for "validator" or similar context?

Reply:
Sir this is autogenerated code from the proto files.